tests/xml2/XMLBearTest.py

```diff
@@ -28,30 +35,15 @@
                            "test_files",
                            "note.xsd")
 
-valid_xml_path = list(open(os.path.join(
-    os.path.dirname(__file__),
-    "test_files",
-    "note.xml"), 'r'))
-
-valid_xml_url = list(open(os.path.join(
-    os.path.dirname(__file__),
-    "test_files",
-    "concept-valid.xml"), 'r'))
-
-invalid_xml_schema = list(open(os.path.join(
-    os.path.dirname(__file__),
-    "test_files",
-    "xsd-error.xml"), 'r'))
-
-invalid_xml_dtd = list(open(os.path.join(
-    os.path.dirname(__file__),
-    "test_files",
-    "dtd-error.xml"), 'r'))
-
-invalid_xml_url = list(open(os.path.join(
-    os.path.dirname(__file__),
-    "test_files",
-    "concept-invalid.xml"), 'r'))
+valid_xml_path = load_testdata("note.xml")
+
+valid_xml_url = load_testdata("concept-valid.xml")
+
+invalid_xml_schema = load_testdata("xsd-error.xml")
+
+invalid_xml_dtd = load_testdata("dtd-error.xml")
+
```

Comment:
I think you can remove these newlines, so we are saving another 5 lines making this commit looking even better :D